tests/benchmark/utils/shared_info.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+
+
+#include <s2n.h>
+#include "tls/s2n_cipher_suites.h"
+#include "tls/s2n_cipher_preferences.h"
+#include <benchmark/benchmark.h>
+#include <stdlib.h>
+#include <string.h>
+#include <cstring>
+#include <string>
+#include <vector>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <sys/ioctl.h>
+#include <sys/param.h>
+#include <poll.h>
+#include <netdb.h>
+#include <stdlib.h>
+#include <signal.h>
+#include <unistd.h>
+#include <string.h>
+#include <stdio.h>
+#include <getopt.h>
+#include <strings.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <netinet/tcp.h>
+#include <iostream>
+#include <sys/stat.h>
+#include <sys/mman.h>
+#include "utils/s2n_safety.h"
+#include "utils/s2n_random.h"
```

Comment:
Sorry you might have gone too far based on the last PR comment. Does this header file need all these headers? The ideal state is a file includes the things it directly needs. When you moved the shared functions in #2996 to a shared C file it made sense to move the headers to the shared spot. Here it looks like you're including all the headers in one spot which isn't the goal of reducing duplicate includes.